cloud/aws/templates/aws_oidc/bin/setup.py

```diff
@@ -1,27 +1,48 @@
-import os
 import subprocess
 import shlex
-import shutil
+from getpass import getpass
+from typing import Dict
 
+from cloud.aws.templates.aws_oidc.bin.aws_cli import AwsCli
 from cloud.aws.templates.aws_oidc.bin.aws_template import AwsSetupTemplate
 from cloud.aws.bin.lib import backend_setup
+from cloud.shared.bin.lib.config_loader import ConfigLoader
+
+# TODO: move these to variable_definitions.json and read docs from there.
```

Comment:
I have a slight preference to mark the TODO against a specific issue (e.g. `TODO(#1234)`) so it's easier to remember what works still needs to be done for a specific issue.

Reply:
Done.